main.js

```diff
@@ -27,6 +34,9 @@ const createWindow = () => {
     mainWindow.show()
```

Comment:
Here is from the #1 https://github.com/lukechu10/Minecraft-Box/issues/1#issue-422048372
```suggestion
    mainWindow.show()
```

So Im hoping this would come in handy